test/backward-stacktrace.h

```diff
@@ -0,0 +1,42 @@
+#include <csignal>
+
+#include "backward.hpp"
+#include "glog/logging.h"
+#include "gtest/gtest.h"
+
+class BackwardStackTrace : public ::testing::Environment {
```

Comment:
We should probably put this in a specific namespace, rather than implicitly adding it to the global namespace: https://google.github.io/styleguide/cppguide.html#Namespaces